src/hooks/useBreakpoints.ts

```diff
@@ -20,6 +20,7 @@ export default function () {
   }, [])
 
   return {
+    badgeNameSize: width > smSize && width < 400,
```

Comment:
The `useBreakpoint` is used to have different screens breakpoints, maybe you can make a different name for this size?